engine/src/build/config/apple/apple_sdk.gni

```diff
@@ -2,16 +2,168 @@
 # Use of this source code is governed by a BSD-style license that can be
 # found in the LICENSE file.
 
+import("//build/toolchain/rbe.gni")
+
+assert(host_os == "mac")
+
+# Common args across all Apple platforms.
 declare_args() {
-  # Path to the Xcode toolchain (XcodeDefault.xctoolchain).
-  apple_host_toolchain_path = ""
+  # The MACOSX_DEPLOYMENT_TARGET variable used when compiling.
+  # Must be of the form x.x.x for Info.plist files.
+  mac_deployment_target = ""
```

Comment:
is this required for ios as well? i guess it's for ipad's "designed for mac"?

Reply:
It's for macOS targets and for host binaries.